src/integration-test/java/se/sundsvall/emailreader/configuration/TestContainersConfiguration.java

```diff
@@ -0,0 +1,19 @@
+package se.sundsvall.emailreader.configuration;
+
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.boot.testcontainers.service.connection.ServiceConnection;
+import org.springframework.context.annotation.Bean;
+import org.testcontainers.containers.MariaDBContainer;
+
+@TestConfiguration(proxyBeanMethods = false)
```

Comment:
You should be able to get rid of this class to if you add
```
#-------------------------------------------------------------------------------
# Datasource settings
#-------------------------------------------------------------------------------
spring.jpa.properties.jakarta.persistence.schema-generation.database.action=validate
# URL to database test container
spring.datasource.url=jdbc:tc:mariadb:10.6.12:///
# Drivers, dialect, etc.
spring.datasource.driver-class-name=org.testcontainers.jdbc.ContainerDatabaseDriver
# Flyway
spring.flyway.enabled=true
```
to your application-it configuration file. If Flyway is used, you can also remove the init-db.sql file (which also will verify that your db scripts works correctly). If you for some reason need to pre-fill your db-model with data, you can add
```
@Sql(scripts = {
	"/path/to/db/scripts/db-script-to-execute.sql"
})
```
below the `@WireMockAppTestSuite` annotation in the IT-testclass.

See api-service-notes for example how its done there (all services maintained by former unmasked-team with db:s have the same pattern).